src/utils.h

```diff
@@ -198,6 +198,7 @@ std::vector<std::pair<std::string, std::string>> get_cgroup_hierarchy_roots();
 std::vector<std::pair<std::string, std::string>> get_cgroup_paths(
     uint64_t cgroupid,
     std::string filter);
+bool is_loaded_module(const std::string &module);
```

Comment:
```suggestion
bool is_module_loaded(const std::string &module);
```

Reply:
Still applies, right?

Reply:
Yep, for sure. Got lost in the formatting fixup that I did after clicking the "Commit suggestion" button. Sorry about that.